### PR TITLE
Cookie permissions dialog

### PR DESCRIPTION
**Pull request by joescottdave**
Adds cookie permissions dialog that will check for a cookie named 'hmlr_policy' with a property 'analytics' property set to true or false.
If no such cookie is present the dialog will appear atop the page, else user permitting the Google Tag Manager will be initialised.

epimorphics/hmlr-linked-data#20

### CODE REVIEW

File: app/views/shared/_cookie_banner.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <h2
+    class="cookie-banner__heading"
+  >
+    This site uses cookies
+  </h2>
+  <p
+    class="cookie-banner__details"
+  >
+    Cookies are files stored on your computer, phone, or tablet.
+    They help us understand how you use this site, like which pages you visit.
```
Comment (ijdickinson) [inline app/views/shared/_cookie_banner.html:15]: I think I'd change "like" to "such as" here

File: app/views/shared/_cookie_banner.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <br/>
+    For more information, view our
+      <a
+        href="#"
```
Comment (ijdickinson) [inline app/views/shared/_cookie_banner.html:19]: I'll ask Dave L where we should point this

**Review by ijdickinson — APPROVED** [review]
Lgtm. I made a couple of minor comments on the way through, and I note that a couple of files have "missing-EOL" sad-faces

File: app/assets/stylesheets/lr_common_styles/_lr-common.scss [hunk 3]
```diff
@@ -4,12 +4,12 @@
 
 /* variables */
 
-$lr-green-dark: #5A8006;
+$lr-green-dark: #5a8006;
```
[the remark this answers is not in the document]
Comment (joescottdave) [reply app/assets/stylesheets/lr_common_styles/_lr-common.scss:7]: Before I worked out that I would have to `bundle update` lr-landing each time I made a change, linting the CSS was a reach in my attempts to debug it.

File: app/assets/javascripts/lr_common_styles/cookie.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
+const HMLR_COOKIE_POLICY = 'hmlr_cookie_policy'
+const COOKIE_DURATION = 365
+
+/**
+ * Retrieve user preferences cookie
+ * If user has accepted analytics, load analytics
+ * If no preference is set, show cookie banner
+ */
+window.onload = function () {
+  var userPreferences = getCookie(HMLR_COOKIE_POLICY)
```
[the remark this answers is not in the document]
Comment (joescottdave) [reply app/assets/javascripts/lr_common_styles/cookie.js:10]: This repo has no eslint config.  As it is supposed to run in Ye Olde Browseres, I thought it'd stick to the basics.

File: app/views/shared/_cookie_banner.html (hunk 2 above)
Comment (joescottdave) [reply app/views/shared/_cookie_banner.html:19]: Will remove the link

**Pull request merged**